spec/controllers/admin_users_controller_spec.rb

```diff
@@ -181,10 +182,47 @@
         sign_in user
 
         expect(LannisterMailer).to_not(receive(:regards_email).with(user, user))
-        get :send_email, params: { id: user.id }
+        get :send_email, params: { id: user.id }, xhr: true
 
         expect(response).to redirect_to(root_path)
       end
     end
   end
+
+  describe '#welcome_email' do
+    context 'admin is signed in' do
+      it 'send email to all users in organization' do
+        organization = create(:organization)
+        admin = create(:admin, email: 'admin@example.com', organization: organization)
+        create(:user, email: 'joe.doe@example.com', organization: organization)
+        create(:user, email: 'alice.doe@example.com')
+        sign_in admin
+
+        perform_enqueued_jobs do
+          get :welcome_email, xhr: true
+
+          mails = ActionMailer::Base.deliveries
+          expect(mails.count).to eq 2
+          expect(mails.map(&:to).flatten).to match_array ['joe.doe@example.com', 'admin@example.com']
+          expect(mails.map(&:to).flatten).to_not match_array ['alice.doe@example.com']
```

Comment:
it'd be better: `expect(mails.map(&:to).flatten).to_not include 'alice.doe@example.com'`